syntaxes/codeblock.json

```diff
@@ -0,0 +1,28 @@
+{
+  "fileTypes": [],
+  "injectionSelector": "L:markup.fenced_code.block.markdown",
+  "patterns": [
+    {
+      "include": "#plantuml-code-block"
```

Comment:
where is this "plantuml" comming from?

Reply:
Sorry about that "plantuml". It was from another snippet. I have pushed an update that does not have it.